legacy_plugins/insight_to_action/scripts/refresh-md.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+// (C) 2022 GoodData Corporation
+
+const dotenv = require("dotenv");
+const path = require("path");
+dotenv.config({ path: path.resolve(__dirname, "../.env") });
+dotenv.config({ path: path.resolve(__dirname, "../.env.secrets") });
+
+const workspace = process.env.WORKSPACE;
+const hostname = process.env.BACKEND_URL;
+const backend = "bear";
```

Comment:
@dremileta this should come from the env variable BACKEND_TYPE with the default being `tiger`.